hugegraph-api/src/main/java/com/baidu/hugegraph/serializer/JsonSerializer.java

```diff
@@ -118,6 +118,7 @@ private String writeIterator(String label, Iterator<?> iter,
                                             iter.getClass());
                 }
                 if (page != null) {
+                    page.replaceAll("[+]", "%2B");
```

Comment:
prefer not transfer just when toJson

Reply:
shall we keep encode `+` to `%2B`?  (just remove it now)